pheweb/load/phenolist.py

```diff
@@ -0,0 +1,90 @@
+#!/usr/bin/env/python3
+
+#phenolist.py pheno-list.json.orig ../r3_1/pheno-list.json TAGLIST_DF4.txt finngen_R4_endpoint_definition.txt R4_COV_PHENO_V1.txt.gz ./generated-by-pheweb | python -m json.tool > pheno-list.json
+
```

Comment:
Maybe add comment what is the use of this file as opposed to almost the same file in /load... how is this used in deploymen etc.

Reply:
Yes will be part of the import workflow